docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -277,24 +407,74 @@ The `locale` prop of the `LocalizationProvider` component have been renamed `ada
  </LocalizationProvider
 ```
 
-## Component slots (custom sub-components)
+## Component slots / component props slots
+
+All the props used to pass props to parts of the UI (e.g: pass a prop to the input) have been replaced by components props slots.
+All the props used to override parts of the UI (e.g: pass a custom day renderer) have been replaced by components slots.
+
+You can find more information about this pattern in the [MUI Base documentation](https://mui.com/base/getting-started/usage/#shared-props).
+
+There changes always apply to all the components that had the prop.
+For example, the `ToolbarComponent` has been replaced by a `Toolbar` component slot on all pickers.
+
+### Input renderer (required in v5)
```

Comment:
Do you think we should consider having a codemod taking care of at least the basic examples in this section?

Reply:
I don't know how much of a pain it would be honestly :grimacing:

Reply:
Well, we can always experiment with it if we'll have the time for it.
Or, get back to it if we'd have a lot of community requests. 🤔